js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts

```diff
@@ -45,16 +45,14 @@ describe("Realm settings general tab tests", () => {
 
     // Enable realm
     realmSettingsPage.toggleSwitch(`${realmName}-switch`);
-    masthead.checkNotificationMessage("Realm successfully updated", true);
+    masthead.checkNotificationMessage("Realm successfully updated");
+    sidebarPage.waitForPageLoad();
 
     // Disable realm
     realmSettingsPage.toggleSwitch(`${realmName}-switch`);
     realmSettingsPage.disableRealm();
     masthead.checkNotificationMessage("Realm successfully updated", true);
```

Comment:
I think we can omit `true` here as it's the default.
```suggestion
    masthead.checkNotificationMessage("Realm successfully updated");
```

Reply:
seems to not work

Reply:
I had that before

Reply:
Should not make a difference, as [it is the default value](https://github.com/keycloak/keycloak/blob/d5bf79b9329e81437d8f0100879e162efa7e2be2/js/apps/admin-ui/cypress/support/pages/admin-ui/Masthead.ts#L97), perhaps it's just flaky?